backend/tests/test_recovery_engine.py

```
@@ -480,7 +504,85 @@ def test_zero_shares_collapses_paths_to_zero_dollars(self):
             current_price=8.0,
             target_yield=0.18,
-            sizing_cap_dollars=5000.0,
+            sizing_cap_pct=25.0,
+            total_capital=20000.0,
         )
         wheel = next(p for p in paths if p["path_id"] == "wheel-cc")
         assert wheel["capital_tied_up"] == pytest.approx(0.0)
+
+
+# ---------------------------------------------------------------------------
+# Issue #234 — sizing_cap_pct × total_capital resolution (W-F)
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)